packages/neos-ui/src/Containers/PrimaryToolbar/MenuToggler/index.js

```diff
@@ -8,6 +8,8 @@ import Button from '@neos-project/react-ui-components/src/Button/';
 import {actions} from '@neos-project/neos-ui-redux-store';
 import {neos} from '@neos-project/neos-ui-decorators';
 
+import logo from './logo.svg';
```

Comment:
I will not couple to logo with the Menu toggler, but introduce a static component that is integrated via the registry. Like this is can be replaced with our extendability features

Reply:
Okay, good idea. There are lots of components that should be refactored ... ;)

Reply:
Done.